docs/modules/elements-color-picker/assets/js/editor/component.md

```diff
@@ -42,5 +42,12 @@ This is the JS component which is responsible for registering the following comm
 
 	removing its `nsEvents` property.
 
+
+- Our initial thought was creating a per-pixel eyedropper, but we dropped this idea (pun intended) since the only way to do that is by
+
+	taking a screenshot of the page and converting to Canvas, which brings a huge performance penalty with it.
+
+	In addition, taking a screenshot of a website is prohibited by the privacy rules of WordPress, so even if the performance would've been OK, we would still have this privacy issue.
+
```

Comment:
Remove. it's in the user browser so there is no privacy issue with it.

Reply:
Done.